cmake/Dependencies.cmake

```diff
@@ -930,12 +933,6 @@ if (CAFFE2_CMAKE_BUILDING_WITH_MAIN_REPO)
     set(ONNX_CUSTOM_PROTOC_EXECUTABLE ${CAFFE2_CUSTOM_PROTOC_EXECUTABLE})
   endif()
   set(TEMP_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
-  # We will build onnx as static libs and embed it directly into the binary.
```

Comment:
This might be what's causing issue with ONNX. Can you change this back?

Reply:
Setting it back leads to error treated as warning (due to /Wx)

>        "D:\pytorch-scripts\caffe2_builders\v140\pytorch\build\Release\INSTALL.vcxproj" (целевой объект по умолчанию) (1) ->
>        "D:\pytorch-scripts\caffe2_builders\v140\pytorch\build\Release\ALL_BUILD.vcxproj" (целевой объект по умолчанию) (3) ->
>        "D:\pytorch-scripts\caffe2_builders\v140\pytorch\build\Release\caffe2\caffe2.vcxproj" (целевой объект по умолчанию) (13) ->
>        "D:\pytorch-scripts\caffe2_builders\v140\pytorch\build\Release\third_party\onnx\onnx.vcxproj" (целевой объект по умолчанию) (20) ->
>        "D:\pytorch-scripts\caffe2_builders\v140\pytorch\build\Release\third_party\onnx\onnx_proto.vcxproj" (целевой объект по умолчанию) (21) ->
>        (Целевой объект ClCompile) ->
>          D:\pytorch-scripts\caffe2_builders\v140\pytorch\build\Release\third_party\onnx\onnx/onnx_onnx_c2.pb.h(39): error C2220: warning treated as error - no 'object' file generated (compi
>        ling source file D:\pytorch-scripts\caffe2_builders\v140\pytorch\build\Release\third_party\onnx\onnx\onnx_onnx_c2.pb.cc) [D:\pytorch-scripts\caffe2_builders\v140\pytorch\build\Releas
>        e\third_party\onnx\onnx_proto.vcxproj]